pkg/infra/iac2/templates/lambda_function/factory.ts

```diff
@@ -9,6 +9,7 @@ interface Args {
     Role: aws.iam.Role
     // envVars: Record<string, pulumi.Output<string>>,
     // dependsOn: []
+    dependsOn: any[]
```

Comment:
Technically, this is `dependsOn?: pulumi.Input<pulumi.Input<pulumi.Resource>[]> | pulumi.Input<pulumi.Resource>`

Reply:
yeah let me go update this